docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -489,15 +489,6 @@ Error: UPGRADE FAILED: Service "external-weblogic-operator-svc" is invalid: spec
 #### Installing an operator and assigning it a service account that doesn't exist
 
 The `helm install` eventually times out and creates a failed release.
-```
-$ helm install kubernetes/charts/weblogic-operator --name op2 --namespace myuser-op2-ns --values o24.yaml --wait --no-hooks
```

Comment:
We probably still need this example command line. The statement before this line is based on the fact that there is  —wait in the command line. Without —wait, there is no time-out.

Reply:
ok. Are suggesting that I put back just the command:
$ helm install kubernetes/charts/weblogic-operator --name op2 --namespace myuser-op2-ns --values o24.yaml --wait --no-hooks
But, **not** the result:
Error: release op2 failed: timed out waiting for the condition
kubectl logs -n kube-system tiller-deploy-f9b8476d-mht6v
...
[kube] 2018/12/06 21:16:54 Deployment is not ready: myuser-op2-ns/weblogic-operator
...
OR, put back both, the command and the result?

Reply:
Yes, let's put the command back, but remove the specific output.

Reply:
Done.